src/main/java/taxi/dao/DriverDaoImpl.java

```diff
@@ -72,13 +74,16 @@ public List<Driver> getAll() {
     public Driver update(Driver driver) {
         String query = "UPDATE drivers "
                 + "SET name = ?, license_number = ? "
+                + "login = ?, password = ? "
                 + "WHERE id = ? AND is_deleted = FALSE";
         try (Connection connection = ConnectionUtil.getConnection();
                 PreparedStatement statement
                         = connection.prepareStatement(query)) {
-            statement.setString(1, driver.getName());
-            statement.setString(2, driver.getLicenseNumber());
-            statement.setLong(3, driver.getId());
+            statement.setLong(1, driver.getId());
+            statement.setString(2, driver.getName());
+            statement.setString(3, driver.getLicenseNumber());
+            statement.setString(4, driver.getLogin());
+            statement.setString(5, driver.getPassword());
```

Comment:
Is it works? First "?" in query it's name, but you set the id. 2, 3, 4, 5 also wrong